python/src/wslink/backends/aiohttp/__init__.py

```diff
@@ -51,6 +51,10 @@ async def _root_handler(request):
 
 
 async def _stop_server(app):
+    if hasattr(app, "stop"):
+        app.stop()
```

Comment:
Should this be `await app.stop()` since it appears `app.stop()` is an async function?

Reply:
probably... ;-)